go1.14/Makefile.common

```diff
@@ -0,0 +1,26 @@
+SELF_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
+include $(SELF_DIR)/../Makefile.common
+
+NAME           := golang-crossbuild
+VERSION        := 1.14.0
+DEBIAN_VERSION ?= 9
+SUFFIX         := -$(shell basename $(CURDIR))
+TAG_EXTENSION  ?=
+
+export DEBIAN_VERSION TAG_EXTENSION
+
+build:
+	@echo ">> Building $(REPOSITORY)/$(NAME):$(VERSION)$(SUFFIX)$(TAG_EXTENSION)"
+	@go run $(SELF_DIR)/../template.go -t Dockerfile.tmpl -o Dockerfile
+	@docker build -t "$(REPOSITORY)/$(NAME):$(VERSION)$(SUFFIX)$(TAG_EXTENSION)" \
+  --build-arg REPOSITORY=$(REPOSITORY) \
+  --build-arg VERSION=$(VERSION) \
+  --build-arg DEBIAN_VERSION=$(DEBIAN_VERSION) \
+  --build-arg TAG_EXTENSION=$(TAG_EXTENSION) \
```

Comment:
Looks like spaces vs. tabs :)

Reply:
@urso I suspect this is an issue in other directories too.  I based the `go1.14` directory on the `go1.13` directory.  I'll review the file in the `go1.14` directory.

```bash
$ diff go1.13/Makefile.common go1.14/Makefile.common
5c5
< VERSION        := 1.13.8
---
> VERSION        := 1.14.0
```

Reply:
I've fixed the spaces vs tabs issue in `go1.14/Makefile.common`.